lib/std/heap/debug_allocator.zig

```diff
@@ -90,11 +90,12 @@ const mem = std.mem;
 const Allocator = std.mem.Allocator;
 const StackTrace = std.builtin.StackTrace;
 
-const default_page_size: usize = @max(std.heap.page_size_max, switch (builtin.os.tag) {
+const default_page_size: usize = switch (builtin.os.tag) {
     .windows => 64 * 1024, // Makes `std.heap.PageAllocator` take the happy path.
     .wasi => 64 * 1024, // Max alignment supported by `std.heap.WasmAllocator`.
-    else => 128 * 1024, // Avoids too many active mappings when `page_size_max` is low.
-});
+    .freestanding => 64 * 1024, // Max alignment supported by `std.heap.WasmAllocator`.
```

Comment:
freestanding does not imply wasm

Reply:
You are seeing my confusion.  I don't understand why having default_page_size be larger than page_size_max would ever be useful.  I agree that freestanding doesn't imply wasm, but then how do I get something compiled as wasm32-freestanding get a 64k default_page_size?